### PR TITLE
alias ugly urls

### PR DESCRIPTION
**Pull request by bjlittle**
This PR makes some minor changes to the rendered URLs for the notebooks in the `README.md`.

### CODE REVIEW

**Comment by bjlittle** [comment]
@dkillick Tweaked a change to fix the travis-ci :wink:

**Comment by dkillick** [comment]
@bjlittle I'm just not sure I like the headings-as-links approach you've proposed here - I think it makes the readme less clear. My preference would be for a sentence under each heading after the description of each course that reads "View the course notebook." Happy to discuss that further though!

File: README.md [hunk 1]
```diff
@@ -4,24 +4,24 @@ Python courses for the scientific researcher
...
 particularly in the fields of oceanography and meteorology.
 
-There are currently five courses:
+There are currently 5 courses:
 
 ### An introduction to numpy
-3.5 hours &mdash; depends on a basic Python background
-http://nbviewer.ipython.org/github/SciTools/courses/blob/master/course_content/notebooks/numpy_intro.ipynb?create=1
+* **3.5 hours** &mdash; depends on a basic Python background
+* [view the course noteboook](http://nbviewer.ipython.org/github/SciTools/courses/blob/master/course_content/notebooks/numpy_intro.ipynb?create=1)
```
Comment (dkillick) [inline README.md]: oooooo 👻 (you've over-o'd "notebook")

File: README.md [hunk 2]
```diff
@@ -4,24 +4,24 @@ Python courses for the scientific researcher
...
 
 ### An introduction to matplotlib
-3 hours &mdash; depends on "An introduction to numpy"
-http://nbviewer.ipython.org/github/SciTools/courses/blob/master/course_content/notebooks/matplotlib_intro.ipynb?create=1
+* **3 hours** &mdash; depends on "An introduction to numpy"
+* [view the course notebook](http://nbviewer.ipython.org/github/SciTools/courses/blob/master/course_content/notebooks/matplotlib_intro.ipynb?create=1)
```
Comment (dkillick) [inline README.md]: Too little punctuation ⁉️ 

How about:
```
... depends on "an introduction to such-and-such." [View the course notebook](http://linky.com).
```

(I've been waiting years for an opportunity to interrobang.)

**Comment by bjlittle** [comment]
@dkillick I think this is now good to go.

File: README.md [hunk 3]
```diff
@@ -4,24 +4,24 @@ Python courses for the scientific researcher
 This repository contains several courses for the benefit of scientific researchers,
 particularly in the fields of oceanography and meteorology.
 
-There are currently five courses:
+There are currently 5 courses:
```
Comment (dkillick) [inline README.md]: Ooh, could you change this back – numbers less than 10 should be rendered as words.

**Comment by dkillick** [comment]
> think this is now good to go

... almost

File: README.md (hunk 3 above)
Comment (bjlittle) [reply README.md]: I really have awoken the beast ...

Comment (dkillick) [reply README.md]: 🐻

**Comment by dkillick** [comment]
🎉

**Comment by bjlittle** [comment]
@dkillick Hoo-rah! Thanks!

**Pull request merged**